### PR TITLE
Fixed accessibility related issues on schedule tab inside ccx dashboard

### PR DESCRIPTION
**Pull request by amir-qayyum-arbisoft**
Hi,

In this PR I fixed issues on schedule tab related to accessibility.

Please review 
@ahsan-ul-haq @muzaffaryousaf @symbolist

### CODE REVIEW

File: lms/templates/ccx/schedule.html [hunk 1]
```diff
@@ -72,16 +72,16 @@
     <h2>${_('Schedule a Unit')}</h2>
     <form role="form" id="add-unit" name="add-unit" class="ccx-form">
       <div class="field">
-        <b>${_('Section')}</b><br/>
-        <select name="chapter"></select>
+        <b id="label_for_section">${_('Section')}</b><br/>
+        <select aria-describedby="label_for_section" name="chapter" role="listbox"></select>
```
Comment (ahsan-ul-haq) [inline lms/templates/ccx/schedule.html]: I guess you probably should use ```<label>``` instead of ```<b>```.

Comment (amir-qayyum-arbisoft) [inline lms/templates/ccx/schedule.html]: they use <b>, i dont think any need to change it, what do u think @symbolist, because i want second opinion.

Comment (symbolist) [inline lms/templates/ccx/schedule.html]: I will defer to @ahsan-ul-haq's opinion.

File: lms/templates/ccx/schedule.html [hunk 2]
```diff
@@ -72,16 +72,16 @@
     <h2>${_('Schedule a Unit')}</h2>
     <form role="form" id="add-unit" name="add-unit" class="ccx-form">
       <div class="field">
-        <b>${_('Section')}</b><br/>
-        <select name="chapter"></select>
+        <b id="label_for_section">${_('Section')}</b><br/>
```
Comment (symbolist) [inline lms/templates/ccx/schedule.html]: Also I think the format `section_label` is preferable to `label_for_section.` Can you update all the ids that you have added in this PR?

File: lms/templates/ccx/schedule.html [hunk 3]
```diff
@@ -57,10 +57,10 @@
   <div class="ccx-sidebar-panel" id="dirty-schedule">
     <h2>${_('Save changes')}</h2>
     <form role="form">
-      <p>${_("You have unsaved changes.")}</p>
+      <p id="hint_save_changes">${_("You have unsaved changes.")}</p>
```
Comment (symbolist) [inline lms/templates/ccx/schedule.html]: "save_changes_hint"

**Comment by symbolist** [comment]
Once you update the 4 ids. :+1:

File: lms/templates/ccx/schedule.html (hunk 1 above)
Comment (amir-qayyum-arbisoft) [inline lms/templates/ccx/schedule.html]: ok,

**Pull request closed without merging**